Examples/Io/Root/src/RootParticleWriter.cpp

```diff
@@ -180,6 +182,7 @@ ActsExamples::ProcessCode ActsExamples::RootParticleWriter::writeT(
       m_pathInX0.push_back(nan);
       m_pathInL0.push_back(nan);
       m_numberOfHits.push_back(-1);
+      m_outcome.push_back(0);
```

Comment:
I think you have to clear `m_outcome` further down ~L212

Reply:
No problem, I should have seen it sooner myself!